code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -80,15 +80,22 @@
 			return 1
 		var/obj/item/clothing/under/U = H.w_uniform
 		if(istype(U))
+			if(user == H)
+				U.attach_accessory(src, user, TRUE)
+				return
 			user.visible_message("<span class='notice'>[user] is putting a [src.name] on [H]'s [U.name]!</span>", "<span class='notice'>You begin to put a [src.name] on [H]'s [U.name]...</span>")
 			if(do_after(user, 40, target=H) && H.w_uniform == U)
```

Comment:
```suggestion
			if(do_after(user, 4 SECONDS, target=H) && H.w_uniform == U)
```
I know this wasn't touched either, but can we change it while we are here?

Reply:
```suggestion
			if(do_after(user, 4 SECONDS, target = H) && H.w_uniform == U)
```